awsutil/client_options_test.go

```diff
@@ -48,7 +48,7 @@ func TestClientOptions(t *testing.T) {
 	})
 	t.Run("Validate", func(t *testing.T) {
 		t.Run("SucceedsWithAllOptionSet", func(t *testing.T) {
-			creds := credentials.NewEnvCredentials()
+			creds := credentials.NewStaticCredentialsProvider("", "", "")
```

Comment:
It seems this is related to [the prior thread](https://github.com/evergreen-ci/cocoa/pull/109#discussion_r1283294795).
It's true that the credentials chain includes checking the environment for credentials.

Reply:
If the creds are not relevant to the test's ability to pass/fail, we can just remove it from this test entirely then rather than initialize empty static creds.

Reply:
removed